apps/ledger-live-mobile/src/components/RootNavigator/PostOnboardingNavigator.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useMemo } from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTheme } from "styled-components/native";
+
+import { ScreenName } from "../../const";
+import { getStackNavigatorConfig } from "../../navigation/navigatorConfig";
+
+import { ParamList } from "../../screens/PostOnboarding/types";
+import PostOnboardingDebugScreen from "../../screens/PostOnboarding/PostOnboardingDebugScreen";
+import PostOnboardingHub from "../../screens/PostOnboarding/PostOnboardingHub";
+import PostOnboardingMockActionScreen from "../../screens/PostOnboarding/PostOnboardingMockActionScreen";
+
+const Stack = createStackNavigator<ParamList>();
+
+const Empty = () => null;
+
+const PostOnboardingNavigator = () => {
+  const { colors } = useTheme();
+  const stackNavigationConfig = useMemo(
+    () => getStackNavigatorConfig(colors, true),
+    [colors],
+  );
+
+  return (
+    <Stack.Navigator
+      screenOptions={{
+        ...stackNavigationConfig,
+        headerShown: true,
+        title: "",
+        headerLeft: Empty,
```

Comment:
`headerLeft` is it mandatory? what impact does this `Empty` have?

Reply:
If you don't specify it there will be a left arrow / "back" arrow by default